engine/src/main/java/io/graphqlcrud/Filters.java

```diff
@@ -0,0 +1,88 @@
+package io.graphqlcrud;
+
+import graphql.Scalars;
+import graphql.schema.*;
+
+public class Filters {
+
+    public static GraphQLInputObjectType.Builder pageInputBuilder() {
+        return GraphQLInputObjectType.newInputObject().name("PageRequest")
+                .field(GraphQLInputObjectField.newInputObjectField().name("limit").type(Scalars.GraphQLInt))
+                .field(GraphQLInputObjectField.newInputObjectField().name("offset").type(Scalars.GraphQLInt));
+    }
+
+    public static GraphQLInputObjectType.Builder filterInputBuilder() {
+        return GraphQLInputObjectType.newInputObject().name("QueryFilter")
+                .field(GraphQLInputObjectField.newInputObjectField().name("id").type(GraphQLTypeReference.typeRef("IDInput")))
+                .field(GraphQLInputObjectField.newInputObjectField().name("title").type(GraphQLTypeReference.typeRef("StringInput")))
+                .field(GraphQLInputObjectField.newInputObjectField().name("clickCount").type(GraphQLTypeReference.typeRef("IntInput")))
```

Comment:
Now using these types add FilterInput on Customer like

```
CustomerFilterInput {
  SSN StringInput,
  ID    IDInput,
  ...
}
```

Reply:
Adding it additionally or modifying the existing ` type Customer` to `type CustomerFilterInput`?

Reply:
As a separate type, see https://graphqlcrud.org/docs/next/find and example of `NoteFilter` and also see at the end of the page